members-only/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,60 @@
+module SessionsHelper
+	def log_in(user)
+    session[:user_id] = user.id
+  end
```

Comment:
Think about using `cookies`
Take a look on this please https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_method_requirement.png

Reply:
We removed sessions and used cookies